bot.go

```diff
@@ -302,11 +302,28 @@ func (bot *BotAPI) Send(c Chattable) (Message, error) {
 	}
 
 	var message Message
-	err = json.Unmarshal(resp.Result, &message)
+	resultStr := string(resp.Result)
+	if resultStr != "true" {
+		err = json.Unmarshal(resp.Result, &message)
+	}
 
 	return message, err
 }
 
+// helper to get raw params
```

Comment:
Comments should be in the form of `name does thing`. 

```suggestion
// PreviewParams lets you preview the parameters that would be sent with a Chattable.
```